test/plugins/wasmedge_zlib/wasmedge_zlib.cpp

```diff
@@ -0,0 +1,277 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "common/defines.h"
+#include "runtime/instance/module.h"
+#include "zlibfunc.h"
+#include "zlibmodule.h"
+
+#include <algorithm>
+#include <array>
+#include <cstdint>
+#include <cstdio>
+#include <gtest/gtest.h>
+#include <string>
+#include <vector>
+
+namespace {
+WasmEdge::Runtime::CallingFrame DummyCallFrame(nullptr, nullptr);
+
+WasmEdge::Runtime::Instance::ModuleInstance *createModule() {
+  using namespace std::literals::string_view_literals;
+  WasmEdge::Plugin::Plugin::load(std::filesystem::u8path(
+      "../../../plugins/wasmedge_zlib/"
+      "libwasmedgePluginWasmEdgeZlib" WASMEDGE_LIB_EXTENSION));
+  if (const auto *Plugin = WasmEdge::Plugin::Plugin::find("wasmedge_zlib"sv)) {
+    if (const auto *Module = Plugin->findModule("wasmedge_zlib"sv)) {
+      return Module->create().release();
+    }
+  }
+  return nullptr;
+}
+
+} // namespace
+
+void fillMemContent(WasmEdge::Runtime::Instance::MemoryInstance &MemInst,
+                    uint32_t Offset, uint32_t Cnt, uint8_t C = 0) noexcept {
+  std::fill_n(MemInst.getPointer<uint8_t *>(Offset), Cnt, C);
+}
+
+static constexpr size_t DATA_SIZE = 1 * 1024 * 1024ULL;
+// static constexpr size_t INPUT_BUFFER_SIZE = 32 * 1024llu;
```

Comment:
Do we need to keep this? If not, please remove the useless comments.

Reply:
I will very probably require this later, so keeping it for now.